lib/dom_utils.coffee

```diff
@@ -167,15 +167,20 @@ DomUtils =
       node = node.parentNode
     false
 
-  # True if element contains the active selection range.
+  # True if element contains the active selection range, or if the element does not support its selection
+  # being accessed.
   isSelected: (element) ->
     if element.isContentEditable
-      node = document.getSelection()?.anchorNode
+      node = document.getSelection().anchorNode
       node and @isDOMDescendant element, node
     else
-      # Note.  This makes the wrong decision if the user has placed the caret at the start of element.  We
-      # cannot distinguish that case from the user having made no selection.
-      element.selectionStart? and element.selectionEnd? and element.selectionEnd != 0
+      try
+        element.selectionStart != element.selectionEnd
```

Comment:
@mrmr1993.  This changes the current behaviour if the selection is a caret/cursor.  E.g. go [here](http://html5doctor.com/demos/forms/forms-example.html), put some text in the billing address like this:

![snapshot](https://cloud.githubusercontent.com/assets/2641335/6242844/4870798e-b72e-11e4-8416-02659b617988.png)

(i.e. with the cursor in the middle of the text), blur, then return via link hints.  The cursor is now at the end.  It also means the cursor ends up in different places depending upon whether we enter the input with `gi` or link hints, which is odd.

Why not just wrap the existing test with a `try`/`catch`?